virtcontainers/monitor.go

```diff
@@ -98,8 +105,8 @@ func (m *monitor) stop() {
 		return
 	}
 
+	m.stopCh <- true
```

Comment:
here I think we should  close stopCh instead of send a value into it, since there will be more than one watchers wait on this channel, and the others wouldn't be notified if we only write once into it.

Reply:
stopCh is internal and only monitor itself listens on it.

Reply:
@bergwolf even though stopCh is internal,  but every watcher's goroutine would select on this channel, so I think it's better to close it to notify those goroutines to terminate.

Reply:
Oh, you are right. I missed the condition check.